Lib/imghdr.py

```diff
@@ -9,6 +9,9 @@
 #-------------------------#
 
 def what(file, h=None):
+    """Return the type of image contained in a file or
+    byte stream.
+    """
```

Comment:
Please adapt PEP 257 in this docstring and the ones below.

In this case, just put everything in the same line and it will be ok.